lib/mixlib/install/product.rb

```diff
@@ -233,56 +246,64 @@ def lookup(key, version = :latest)
   end
 
   product "marketplace" do
-    product_name "Chef Cloud Marketplace addon"
-    package_name "chef-marketplace"
-    ctl_command "chef-marketplace-ctl"
-    config_file "/etc/chef-marketplace/marketplace.rb"
+    product_name    "Chef Cloud Marketplace addon"
+    package_name    "chef-marketplace"
+    omnibus_project "chef-marketplace"
+    ctl_command     "chef-marketplace-ctl"
+    config_file     "/etc/chef-marketplace/marketplace.rb"
   end
 
   product "omnibus-toolchain" do
-    product_name "Omnibus Toolchain"
-    package_name "omnibus-toolchain"
+    product_name    "Omnibus Toolchain"
+    package_name    "omnibus-toolchain"
+    omnibus_project "omnibus-toolchain"
   end
 
   product "private-chef" do
-    product_name "Enterprise Chef (legacy)"
-    package_name "private-chef"
-    ctl_command "private-chef-ctl"
-    config_file "/etc/opscode/private-chef.rb"
+    product_name    "Enterprise Chef (legacy)"
+    package_name    "private-chef"
+    omnibus_project "private-chef"
+    ctl_command     "private-chef-ctl"
+    config_file     "/etc/opscode/private-chef.rb"
   end
 
   product "push-jobs-client" do
     product_name "Chef Push Server"
+    omnibus_project "push-jobs-client"
     package_name do |v|
       v < version_for("1.3.0") ? "opscode-push-jobs-client" : "push-jobs-client"
     end
```

Comment:
Ditto on here. The omnibus project name changed at the same time as the package name.